# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mine/sanitize-retry-skip.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Mark the default choise more obviously front (also alphabetically -
beside the Eject case).

  - Rene Rebe <rene@exactcode.de>

diff -ur mine-0.23-vanilla/gasgui.c mine-0.23/gasgui.c
--- mine-0.23-vanilla/gasgui.c	2005-03-23 09:51:06.000000000 +0100
+++ mine-0.23/gasgui.c	2006-05-29 16:22:40.000000000 +0200
@@ -133,7 +133,7 @@
 			sourcedir, config, p->name);
 		printf("Not found on webserver: %s\n\n", p->name);
 		while (1) {
-			printf("Skip or Retry (default=Retry) > ");
+			printf("Retry or Skip (default=Retry) > ");
 			fflush(stdout);
 			fgets(command, 1024, stdin);
 			if ( command[0] == 0 || command[0] == '\n' ||
@@ -164,7 +164,7 @@
 		printf("Please insert Disk #%d and press ENTER.\n\n",
 		       p->disk_number);
 		while (1) {
-			printf("Skip, Retry or Eject disk (default=Retry) > ");
+			printf("Retry, Eject or Skip disk (default=Retry) > ");
 			fflush(stdout);
 			fgets(command, 1024, stdin);
 			if ( command[0] == 0 || command[0] == '\n' ||
@@ -212,7 +212,7 @@
 		if ( rc ) {
 			printf("Error while installing package %s!\n\n", p->name);
 			while (1) {
-				printf("Skip or Retry (default=Retry) > ");
+				printf("Retry or Skip (default=Retry) > ");
 				fflush(stdout);
 				fgets(command, 1024, stdin);
 				if ( command[0] == 0 || command[0] == '\n' ||
